docs/platforms/powershell/data-management/data-collected.mdx

```diff
@@ -0,0 +1,79 @@
+
+To send all HTTP headers, set `$_.SendDefaultPii = true` in the `Start-Sentry` command.
+
+Additionally a [data scrubber](/platforms/powershell/data-management/sensitive-data/) removes sensitive data from headers (and a lot of other fields) right before sending data to Sentry.
+
+## Cookies
+
+By default, the Sentry SDK doesn't send cookies. Sentry tries to remove any cookies that contain sensitive information, such as the Session ID and CSRF Token cookies in Django.
+
+If you want to send cookies, set `$_.SendDefaultPii = true` in the `Start-Sentry` command.
+
+Additionally a [data scrubber](/platforms/powershell/data-management/sensitive-data/) removes sensitive data from cookies (and a lot of other fields) right before sending data to Sentry.
+
+## Information About Logged-in User
+
+By default, the Sentry SDK doesn't send any information about the logged-in user, such as email address, user ID, or username. Even if enabled, the type of logged-in user information you'll be able to send depends on the integrations you enable in Sentry's SDK. Most integrations won't send any user information. Some will only set the user ID, but there are a few that will set the user ID, username, and email address.
+
+To start sending logged-in user information, set `$_.SendDefaultPii = true` in the `Start-Sentry` command.
+
+## Users' IP Address
+
+By default, the Sentry SDK doesn't send the user's IP address. Even if enabled, whether you're able to send the user's IP address or not, will depend on the integrations you enable in Sentry's SDK. Most integrations won't set the user's IP address at all.
+
+To enable sending the user's IP address, set `$_.SendDefaultPii = true` in the `Start-Sentry` command.
+
+## Request URL
+
+The full request URL of outgoing and incoming HTTP requests is **always sent to Sentry**. Depending on your application, this could contain PII data.
+
+## Request Query String
+
+The full request query string of outgoing and incoming HTTP requests is **always sent to Sentry**. Depending on your application, this could contain PII data.
+
+## Request Body
+
+The request body of incoming HTTP requests can be sent to Sentry. Whether it's sent or not, depends on the type and size of request body as described below:
+
+- **The type of the request body:**
+  - JSON and form bodies are sent
+  - Raw request bodies are always removed
+  - Uploaded files in the request bodies are never sent to Sentry
+- **The size of the request body:** There's a ["max_request_body_size"](/platforms/python/configuration/options/#max-request-body-size) option that's set to `medium` by default. This means that larger request bodies aren't sent to Sentry.
+
+If you want to prevent bodies from being sent to Sentry altogether, set `max_request_body_size` to `"never"`.
```

Comment:
This is server-side SDK stuff so not related to PowerShell (or .net in general for that matter, only asp.net)

```suggestion
```

Reply:
I've made the Request Body stuff platform specific (ASP.NET and ASP.NET Core only). 

I think the Request URL and Query string are relevant for other integrations though, since we [capture this information on outbound HTTP requests](https://github.com/getsentry/sentry-dotnet/blob/dc3c0dd39fe5f593e9d43ed0d6fbc7a7b42436ce/src/Sentry/SentryHttpMessageHandler.cs#L70-L73) via the `SentryHttpMessageHandler`.

That may be a bit of an edge case in Powershell...